_data/tabs.yml

```diff
@@ -1,8 +1,8 @@
 - name: Home
   link: /
 
-- name: Announcements
-  link: /announcements/
+#- name: Announcements
+#  link: /announcements/
```

Comment:
I suggest leaving this line blank for readability. The history exists in version control so there's no need to keep it in the file.

```suggestion
```